cinema_service/settings.py

```diff
@@ -125,7 +126,7 @@
 
 USE_I18N = True
 
-USE_TZ = False
+USE_TZ = True
```

Comment:
The USE_TZ setting is set to True, which is good for enabling timezone support in Django.